actor-typed/src/main/scala/org/apache/pekko/actor/typed/ActorRef.scala

```diff
@@ -49,8 +49,8 @@ trait ActorRef[-T] extends RecipientRef[T] with java.lang.Comparable[ActorRef[_]
   /**
    * Unsafe utility method for widening the type accepted by this ActorRef;
    * provided to avoid having to use `asInstanceOf` on the full reference type,
-   * which would unfortunately also work on non-ActorRefs. Use it with caution,it may cause a [[ClassCastException]] when you send a message
-   * to the widened [[ActorRef[U]]].
+   * which would unfortunately also work on non-ActorRefs. Use it with caution,it may cause a [[java.lang.ClassCastException]] when you send a message
+   * to the widened [[ActorRef ActorRef[U]]].
```

Comment:
this seems invalid?

Reply:
It's seems invalid.
Due to `[[ActorRef[U]]]`, following warn log is appeared:
```
[warn] /path/actor-typed/src/main/scala/org/apache/pekko/actor/typed/ActorRef.scala:49:3: Could not find any member to link for "ActorRef[U".
```